.github/workflows/ci.yml

```diff
@@ -5,34 +5,20 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
     - uses: actions/checkout@v1
-    - name: Install wget and configure Microsoft apt repository
-      run: |
-        sudo apt install -y wget apt-transport-https
-        wget -q https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
-        sudo dpkg -i packages-microsoft-prod.deb
-        sudo add-apt-repository universe
-        sudo apt update
-    - name: install .Net Core 2.1
-      run: |
-        sudo apt install dotnet-sdk-2.1
-    - name: Nuke NSec and Secp256k1.Net from source tree to make sure we don't depend on them
+    - name: Remove NSec and Secp256k1.Net from source tree to make sure we don't depend on them
       run: rm -rf src/NSec src/Secp256k1.Net
+    - name: Remove NET Core 3 (v2.x will be still available)
+      run: sudo apt remove dotnet-{sdk,runtime,runtime-deps}-3.0
```

Comment:
oh, forgot, it's better to use `purge` than `remove`

Reply:
that only makes a difference for configuration files, and i don't even know if Net Core 3 has any. do you really want me to change this?